core/web/presenters/job.go

```diff
@@ -175,29 +177,33 @@ func NewOffChainReportingSpec(spec *job.OffchainReportingOracleSpec) *OffChainRe
 
 // OffChainReporting2Spec defines the spec details of a OffChainReporting2 Job
 type OffChainReporting2Spec struct {
-	ContractAddress                        ethkey.EIP55Address  `json:"contractAddress"`
-	P2PBootstrapPeers                      pq.StringArray       `json:"p2pBootstrapPeers"`
-	IsBootstrapPeer                        bool                 `json:"isBootstrapPeer"`
-	EncryptedOCRKeyBundleID                null.String          `json:"keyBundleID"`
-	TransmitterAddress                     *ethkey.EIP55Address `json:"transmitterAddress"`
-	ObservationTimeout                     models.Interval      `json:"observationTimeout"`
-	BlockchainTimeout                      models.Interval      `json:"blockchainTimeout"`
-	ContractConfigTrackerSubscribeInterval models.Interval      `json:"contractConfigTrackerSubscribeInterval"`
-	ContractConfigTrackerPollInterval      models.Interval      `json:"contractConfigTrackerPollInterval"`
-	ContractConfigConfirmations            uint16               `json:"contractConfigConfirmations"`
-	CreatedAt                              time.Time            `json:"createdAt"`
-	UpdatedAt                              time.Time            `json:"updatedAt"`
+	ContractID                             null.String     `json:"contractID"`
+	Relay                                  relay.Network   `json:"relay"`
+	RelayConfig                            models.JSON     `json:"relayConfig"`
```

Comment:
`models.JSON` is kinda a shitty type.

Prefer `datatypes.JSON` if you can

Reply:
We should axe `models.JSON` altogether

Reply:
Agree with @archseer and created a card to track here:

https://app.shortcut.com/chainlinklabs/story/23678/remove-models-json-type

Reply:
moved to a map[string]interface{} so we can use a toml heading for UI clarify